modules/azure/api_management_api_schema/variables.tf

```diff
@@ -0,0 +1,19 @@
+variable "resource_group_name" {
+  type        = string
+  description = "Name of the resource group."
+}
+
+variable "api_management_name" {
+  type        = string
+  description = "The name of the API management service."
+}
+
+variable "schema" {
```

Comment:
You may want to consider splitting this object variable up into separate variables for readability:
- schema_file_location
- schema_type
- schema_id

Reply:
Ok, you are right, splitted